src/containers/SignUpForm.tsx

```diff
@@ -27,7 +27,7 @@ const SignUpForm = () => {
                 return;
               }
             }
-            history.push("/success");
+            history.push("../success");
```

Comment:
use relative links, but that might be wrong... might need to be . now that i'm seeing this